.github/workflows/jenkins-x/changelog.sh

```diff
@@ -13,9 +13,7 @@ echo "setting git user"
 git config --global user.name jenkins-x-bot-test
 git config --global user.email "jenkins-x@googlegroups.com"
 
-git add * || true
-git commit -a -m "chore: release $VERSION" --allow-empty
-git tag -fa v$VERSION -m "Release version $VERSION"
+git tag -fa v$VERSION -m "chore: release version $VERSION" HEAD
```

Comment:
Why head at the end?

Reply:
I was trying to make it obvious that you tag the last commit and if you want to go back you can do `HEAD~1`

Reply:
It's the default, we can skip imo.